content/800-data-platform/200-projects/100-create.mdx

```diff
@@ -0,0 +1,152 @@
+    <Admonition>
+
+    **Note**<br /><br />
+    For more information, see [Static egress IP address](/data-platform/static-ips). 
+    
+    </Admonition>
+5. From **Choose a Data Proxy region**, select the geographic location for the Data Proxy.
+    <Admonition>
+    
+    **Note**<br /><br />
+    Select a Data Proxy location that is close to the geographic location of your database.<br /><br />
+    If you select a location that is remote from your database, then when your application sends requests to your database through the Data Proxy, you might notice some delay in the responses.
+    </Admonition>
+6. Click **Create project**.
+    <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1669396285/docs/pdp/create-manage-projects/create-project-01-paste-connection-string-select-data-proxy-region-enable-static-ips_p5wauf.png"
+        alt="Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs"
+        width="500px"
+    />
+
+### Result
+
+The platform creates your project and assigns it a random name, such as **Grey&nbsp;fish&nbsp;<random-code\>**, where **<random-code\>** is an auto-generated code.
+<br />
+
+**If your database does not contain any models (tables or collections)**
+
+You see a notification that your database is empty. Because your database does not contain any models, the introspection cannot generate a Prisma schema.
+
+Later, when you open the [Data Browser](/data-platform/data-browser) or [Query Console](/data-platform/query-console), each indicates that your project is missing a Prisma schema.
+<br />
+
+**If your database contains models**
+
+The platform introspects the database to read the data models so that it can generate the Prisma schema. The Prisma schema enables the features in the [Data Browser](/data-platform/data-browser), [Query Console](/data-platform/query-console), and [Schema Viewer](/data-platform/schema-viewer), where you can edit and query your data, or view your schema.
+
+
+### What's next
+
+You can [develop your Prisma schema](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model) further and [migrate the schema changes you make to your database](https://www.prisma.io/docs/concepts/components/prisma-migrate) with `prisma migrate dev`. You can then push the changes to a GitHub repository. 
+
+With this, you can [enable schema synchronization](#optional-enable-schema-synchronization) so that the platform can read the schema from your repository and update your project with the latest schema and keep the models and attributes up-to-date when you use [Data Browser](/data-platform/data-browser), [Query Console](/data-platform/query-console), and [Schema Viewer](/data-platform/schema-viewer).
```

Comment:
> You can then push the changes to a GitHub repository.

What about: If you push your schema to GitHub repository, you can enable schema synchronization?

Im aiming for something more conditional rather than action "You can then.."